Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.9/x64/lib/python3.8/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
fig, axs = plt.subplots(nrows=1, ncols=2, figsize=(10, 5))
hmn = small_binary_heatmap(humans[0], ax=axs[0], legend=False, title="One Human Brain Network")
hma = small_binary_heatmap(aliens[0], ax=axs[1], legend=False, title="One Alien Brain Network")

add_binary_cbar(humans[0])
plt.tight_layout()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-11-435f874ed28f>[0m in [0;36m<module>[0;34m[0m
[1;32m      3[0m [0mhma[0m [0;34m=[0m [0msmall_binary_heatmap[0m[0;34m([0m[0maliens[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m,[0m [0max[0m[0;34m=[0m[0maxs[0m[0;34m[[0m[0;36m1[0m[0;34m][0m[0;34m,[0m [0mlegend[0m[0;34m=[0m[0;32mFalse[0m[0;34m,[0m [0mtitle[0m[0;34m=[0m[0;34m"One Alien Brain Network"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0;34m[0m[0m
[0;32m----> 5[0;31m [0madd_binary_cbar[0m[0;34m([0m[0mhumans[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      6[0m [0mplt[0m[0;34m.[0m[0mtight_layout[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'add_binary_cbar' is not defined
NameError: name 'add_binary_cbar' is not defined

